code/src/store/api/appApi.ts

```diff
@@ -5,26 +5,29 @@ import {
   createApi,
   fetchBaseQuery
 } from "@reduxjs/toolkit/query/react";
-import { LOCAL_STORAGE_KEYS } from "@/constants/common";
 import { apiNames } from "@/store/constants";
+import { RootState } from "@/store/store";
 
 export type SnackbarPayload = {
   isSnackbarHidden?: boolean;
 };
 
 type CustomFetchBaseQueryError = FetchBaseQueryError & SnackbarPayload;
 
-const baseQuery: BaseQueryFn<
+type CustomBaseQuery = BaseQueryFn<
```

Comment:
It would be better to change naming to BaseQuery type, the same for the error

Reply:
Fixed as you proposed